src/test/ui/consts/min_const_fn/min_const_fn.rs

```diff
@@ -97,10 +97,6 @@ const fn foo30_2(x: *mut u32) -> usize { x as usize }
 //~^ ERROR casting pointers to ints is unstable
 const fn foo30_2_with_unsafe(x: *mut u32) -> usize { unsafe { x as usize } }
 //~^ ERROR casting pointers to ints is unstable
-const fn foo30_4(b: bool) -> usize { if b { 1 } else { 42 } }
```

Comment:
Why remove these?

Reply:
The errors from the HIR checker are emitted early enough that the MIR checker never runs, and all other errors in this file are never emitted.

Reply:
I see; makes sense 👍